angrmanagement/ui/main_window.py

```diff
@@ -13,11 +13,13 @@
 from .menus.file_menu import FileMenu
 from .menus.analyze_menu import AnalyzeMenu
 from .menus.help_menu import HelpMenu
+from .menus.view_menu import ViewMenu
 from ..config import IMG_LOCATION
 from .workspace import Workspace
 from .dialogs.load_binary import LoadBinary, LoadBinaryError
 from .dialogs.new_state import NewState
 from .toolbars import StatesToolbar, AnalysisToolbar, FileToolbar
+from functools import partial
```

Comment:
Nitpick: Please move this import line to the first group of import (together with `import pickle` and `import os`) since `functools` is a Python default package, just like `pickle` and `os`.